Directory.Packages.props

```diff
@@ -67,7 +66,6 @@
     <PackageVersion Include="Microsoft.VisualStudio.ProjectSystem" Version="17.4.221-pre" />
     <PackageVersion Include="Microsoft.VisualStudio.ProjectSystem.Managed.VS" Version="17.2.0-beta1-20502-01" />
     <PackageVersion Include="Microsoft.VisualStudio.ProjectSystem.VS" Version="17.4.221-pre" />
-    <PackageVersion Include="Microsoft.VisualStudio.SolutionPersistence" Version="$(MicrosoftVisualStudioSolutionPersistenceVersion)" />
```

Comment:
Can Microsoft.VisualStudio.SolutionPersistence be removed from our NuGet.Config and Directory.Packages.Props, now? I see it was added recently for this command in https://github.com/NuGet/NuGet.Client/pull/6219/files, so not sure if this was the only reason it was brought into the repo and the `SourceBuildPrebuiltBaseline` as well.